### ISSUE TITLE
Upload 0.12.0 sdist?

### ISSUE THREAD

**Issue by rpkilby** [carltongibson/django-filter#387]
Could `sdist`s and `bdist_wheel`s both be upload to pypi? We have a package that depends on django-filter 0.12.0 (no sdist currently) however tests are executed via `python setup.py test`. setuptools is unable to parse wheel distributions, so fails when installing test dependencies.

- sdist should allows setuptools to function
- pip preferences wheels over sdists anyway

**Comment by carltongibson** [comment]
Grrr! I've done this a million times. 

(Something is broken. I don't know what... — let me try again.)

**Comment by rpkilby** [comment]
While you're at it - think I could guilt you into releasing 0.13 too? ;)

**Comment by carltongibson** [comment]
:-) Already on it...

**Comment by rpkilby** [comment]
Many thanks

**Comment by carltongibson** [comment]
https://pypi.python.org/pypi/django-filter
